### PR TITLE
1.0.x #433

### PR DESCRIPTION
**Pull request by lenglund**
Here's the branch with the style guide-related changes. There were a lot of tweaks, and I assume it will take some time to get this merge-ready. Thanks in advance for the work! 

I'm not entirely done with the style guide issue (#433), but this is a good start to roll out with version 1.0. I've logged additional issues for 1.1 that will expand on this (primarily need to add error handling conventions...and things I've no doubt missed).

### CODE REVIEW

File: compair/static/modules/login/login-partial.html [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
     <div ng-show="!showCreateUserForm">
-        <h2 class="text-center">Welcome to ComPAIR</h2>
+        <h1 class="text-center">Welcome to ComPAIR</h1>
         <div class="row">
             <form name="loginForm" class="form col-sm-6" ng-submit="submit()" novalidate>
                 <fieldset>
                     <div ng-if="AuthTypesEnabled.cas">
-                        <h3>Log in with your CWL here:</h3>
-                        <a id="cwl_login" href="/api/cas/login">
-                            <img class="center-block" src="https://www.auth.cwl.ubc.ca/CWL_login_button.gif" width="76" height="25" alt="CWL Login" border="0" valign="middle">
```
Comment (andrew-gardener) [inline compair/static/modules/login/login-partial.html]: I believe that it is a requirement to use the CWL image

Comment (andrew-gardener) [inline compair/static/modules/login/login-partial.html]: added CWL image back for now https://github.com/ubc/compair/pull/558/commits/1c1f7c8d1030e35b9598d680ce270b89c3461064

**Comment by lenglund** [comment]
@xcompass - Do I recall a conversation about the CWL image not being required anymore or did I make this up? =)

**Comment by xcompass** [comment]
It is still on their checklist, but I remember hearing it is not required if it doesn't fit on the page.

**Comment by lenglund** [comment]
Sounds like the re-add is good then, @andrew-gardener, thanks!

**Comment by xcompass** [comment]
👍

**Pull request merged**